src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -97,15 +97,15 @@ export class CourseDetailBarChartComponent implements OnChanges {
 
     private createLabels() {
         const prefix = this.translateService.instant('calendar_week');
-        const startDate = moment().subtract(3 + 4 * -this.currentPeriod, 'weeks');
-        const endDate = this.currentPeriod !== 0 ? moment().subtract(4 * -this.currentPeriod, 'weeks') : moment();
+        const startDate = moment().subtract(15 + 16 * -this.currentPeriod, 'weeks');
+        const endDate = this.currentPeriod !== 0 ? moment().subtract(16 * -this.currentPeriod, 'weeks') : moment();
         let currentWeek;
-        for (let i = 0; i < 4; i++) {
+        for (let i = 0; i < 16; i++) {
             currentWeek = moment()
-                .subtract(3 + 4 * -this.currentPeriod - i, 'weeks')
+                .subtract(15 + 16 * -this.currentPeriod - i, 'weeks')
```

Comment:
Could we extract these values into constants for it, instead of having hardcoded numbers? It think this would add more clarity to the code.
You already did something similar for the .java files → https://github.com/ls1intum/Artemis/pull/3817/files#diff-c8981cc5e7f04e7084e297989da9a545bdc3d79c630b484bdcfcbfafaf26bea2R318

Reply:
Yes this would actually improve the comprehension of the values and makes it easier to change them in the future :) 👍  9472a29